File: CiscoPCCE.Toolkit/Bean/MediaConfig.cs
using System.Xml.Serialization;

using CiscoPCCE.Toolkit.Sdk;

namespace CiscoPCCE.Toolkit.Bean
{
    [XmlType("media")]
    public class MediaConfig : BaseApiBean
    {
        [XmlElement("codec")]
        public string? Codec { get; set; }

        [XmlElement("mRCPVersion")]
        public string? MRCPVersion { get; set; }

        [XmlElement("overrideSystemPrompt")]
        public bool? OverrideSystemPrompt { get; set; }
    }
}

File: toolkit/bean/MediaConfig.java
// ----------------------------------------------
package com.cisco.ccbu.cce.unifiedconfig.toolkit.bean;

import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiBean;
import javax.xml.bind.annotation.XmlSeeAlso;
import javax.ws.rs.Path;
import javax.xml.bind.annotation.XmlRootElement;
import java.util.List;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.ReferenceBean;
import java.util.Date;
import com.cisco.ccbu.cce.unifiedconfig.toolkit.BaseApiListBean;

import javax.xml.bind.annotation.XmlElement;

import javax.xml.bind.annotation.XmlElementWrapper;


@XmlRootElement(name="media")
public class MediaConfig extends BaseApiBean {
  private String codec;
  private String mRCPVersion;
  private Boolean overrideSystemPrompt;

  public String getCodec() {
     return this.codec;
  }

  public void setCodec(String codec) {
     this.codec = codec;
  }

  public String getMRCPVersion() {
     return this.mRCPVersion;
  }

  public void setMRCPVersion(String mRCPVersion) {
     this.mRCPVersion = mRCPVersion;
  }

  public Boolean getOverrideSystemPrompt() {
     return this.overrideSystemPrompt;
  }

  public void setOverrideSystemPrompt(Boolean overrideSystemPrompt) {
     this.overrideSystemPrompt = overrideSystemPrompt;
  }


}
